plc4j/integrations/apache-nifi/nifi-plc4x-processors/src/main/java/org/apache/plc4x/nifi/Plc4xSinkProcessor.java

```diff
@@ -64,12 +67,13 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
 
             if (tags != null){
                 for (Map.Entry<String,PlcTag> tag : tags.entrySet()){
-                    builder.addTag(tag.getKey(), tag.getValue());
+                    builder.addTag(tag.getKey(), tag.getValue(), flowFile.getAttribute(tag.getKey()));
```

Comment:
What attribute is this???  Isn't this a breaking change?

Reply:
There is no @ReadsAttribute entry for anything new

Reply:
Also, what if it is not present?

Reply:
This flowfile's attribute is the value we need to write into the Plc. I forgot including this on the previous PR.
I have added a check to see if it is present. Similar to Plc4xSinkRecordProcessor and record fields.